src/components/shared/team/ProfileCard.jsx

```diff
@@ -2,16 +2,19 @@ import React from 'react';
 import './/profileCard.css';
 
 function ProfileCard({ img, name, title }) {
+  const itemsDimension = "w-[calc(9.8rem*0.8)] h-[calc(10.1rem*0.8)] md:w-[9.8rem] md:h-[10.1rem]"
   return (
     <>
-      <div className= 'flex w-[23.8rem] h-[13.0rem] bg-[rgb(43,57,80)] rounded-[1.25rem] px-[0.8rem] py-[1.5rem] gap-[1.25rem]'>
-        <div className='w-[9.8rem] h-[10.1rem] rounded-[5.7rem] overflow-hidden aspect-auto'>
+      <div className= 'flex w-[calc(23.8rem*0.8)] h-[calc(13.0rem*0.8)] md:w-[23.8rem] md:h-[13.0rem] bg-[rgb(43,57,80)] rounded-[1.25rem] px-[0.8rem] py-[1.5rem] gap-[1.25rem]'>
+        <div className={itemsDimension + ' rounded-[5.7rem] overflow-hidden aspect-auto'}>
           <img className='object-cover w-full h-full' src={img}></img>
         </div>
-        <div className='h-[10.1rem] bg-transparent flex flex-col gap-[2.3rem]'>
+        <div className={itemsDimension + ' bg-transparent flex flex-col gap-[calc(2.3rem*0.3)] md:gap-[2.3rem]'}>
           <h3 className='text-h3 font-header bg-transparent'>{name}</h3>
           <p className='font-content bg-transparent text-[0.81rem]'>{title}</p>
-          <p className='font-content bg-transparent text-[0.81rem]'>Email: </p>
+          <p className='font-content bg-transparent text-[0.81rem] hidden md:block'>Email: </p>
+          {/* Mobile */}
+          <p className='font-content bg-transparent text-[0.81rem] block md:hidden'>Example@gmail.com</p>
```

Comment:
just wondering once we have a common email between departments how were you plannign to handle it? are you going to have a dict e.g. statically mapping tabs to emails?

Reply:
I haven't thought about it much, but yeah that could work since I'm keeping the current department anyway.